core/src/server/rpc_module.rs

```diff
@@ -485,7 +484,71 @@ impl<Context> From<RpcModule<Context>> for Methods {
 }
 
 impl<Context: Send + Sync + 'static> RpcModule<Context> {
+	/// Register a low-level method call where
+	/// it's possible to get enforce order of things
+	/// on the connection such as ordering that response
+	/// is sent before a subscription notifcation or something similar.
+	///
+	/// ## Examples
+	///
+	/// ```
+	/// use jsonrpsee_core::server::{RpcModule, MethodResponse, response_channel};
+	/// use jsonrpsee_types::ResponsePayload;
+	/// use futures_util::FutureExt;
+	///
+	/// let mut module = RpcModule::new(());
+	/// module.register_raw_method("say_hello", |id, _params, _ctx, max_response_size| {
+	///    let (tx, rx) = response_channel();
+	///
+	///    // This future will be spawned after the method call has been
+	///    // sent out on the socket message buffer.
+	///    // This may useful if one needs to order operations in some manner.
+	///    tokio::spawn(async move {
+	///         // Wait for response to sent to the internal WebSocket message buffer
+	///        // and if that fails just quit because it means that the connection
+	//         // was already closed.
+	///        if rx.is_sent().await.is_err() {
+	///           return;
+	///        }
+	///
+	///        loop {
+	///           tokio::time::sleep(std::time::Duration::from_millis(100)).await;
+	///           println!("after method response");
+	///       }
+	///    });
+	///
+	///    MethodResponse::response(id, ResponsePayloadV2::result("foo"), max_response_size).notify_on_success(tx)
+	/// }).unwrap();
+	///```
+	pub fn register_raw_method<F>(
```

Comment:
this doesn't bring much functionality rather than access to all data such as raw JSON params and ID, not sure if we need it

Reply:
I'd probably not bother adding it for now because the .notify_on_ API does what we need anyways

If we do keep it, then the comment should prob be tweaked, since "it's possible to get enforce order of things on the connection" isn't unique to this any more now anyways :)